app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -47,7 +48,64 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              rows={4}
+            />
+            <Select
+              isClearable
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contacts"
+              name="ValueList"
+              placeholder="Select assigned contact"
+              options={[
+                {
+                  label: "Olive",
+                  value: "oliver",
+                },
+                {
+                  label: "James",
+                  value: "james",
+                },
+                {
+                  label: "Emily",
+                  value: "emily",
+                },
+              ]}
+            />
+            <Select
+              isMulti
+              required
+              className="w-full flex-grow-0"
+              label="Tags"
+              name="ValueList"
```

Comment:
How does it make sense to provide the same `name` for both the Select components? Please understand what Formik does. It seems to me you haven't understood Formik very well.

Also you should've caught this in your self-review.

Reply:
Actually sorry for abstrubly closing this request.
I will explain my points as mentioned below one by one
- This commit is only for setting up a project and mistakenly added UI for note create note UI pane 
- I have gone through and revised Yup and Formik 
- I have implemented all the mentioned changes in my next branch #2 that's why I closed this one
- I thought replying to this will cause unnecessarily notification,   that's why  I haven’t replied but from next time whenever you ask questions I will reply then and there
- This is my mistake I won't repeat this again
-I had raise PR for #2 with suggested review changes.

Reply:
Understood. Thanks for clarifying.